instructor/common.py

```diff
@@ -195,6 +195,18 @@ def assignments_edit(request, assignment_pk):
 
     return redirect('common_assignment_modify')
 
+
+def check_assignment_owner(func):
+    def check_owner(request):
```

Comment:
The decorator looks good overall. You should also use `functools.wraps` to ensure that you retain the name and docstring of the function being decorated: http://stackoverflow.com/questions/308999/what-does-functools-wraps-do